tools/swift_stdlib_tool/swift_stdlib_tool.py

```diff
@@ -28,10 +29,21 @@ def _copy_swift_stdlibs(binaries_to_scan, sdk_platform, destination_path):
   """Copies the Swift stdlibs required by the binaries to the destination."""
   # Rely on the swift-stdlib-tool to determine the subset of Swift stdlibs that
   # these binaries require.
+  _, stdout, _ = execute.execute_and_filter_output(
+      ["xcode-select", "--print-path"], raise_on_failure=True)
+
+  developer_dir = stdout.strip()
```

Comment:
Should we get this from the `DEVELOPER_DIR` environment? That would save us one `xcode-select` invocation.

```suggestion
  developer_dir = os.environ["DEVELOPER_DIR"]
```

Reply:
oops sorry, race. Is that one always going to match the format we need here? I've noticed that xcrun happily accepts the .app dir, or the .app/Contents/Developer dir, at the very least calling that above + this var being set, will normalize that

Reply:
ah I just realized this is a cherry pick, nevermind.

(`DEVELOPER_DIR` is the one injected by Bazel, so it should be the .app/Contents/Developer.)

Reply:
but if you launch bazel with that set to one without it, does it rewrite it?

Reply:
Hmm, I haven't tried that 🤔. I see that wrapped_clang and swift worker use it now, seems like bazel does rewrite it.

Reply:
yea looks like it gets normalized https://github.com/bazelbuild/rules_apple/pull/1356